http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcJsonConfiguration.java

```diff
@@ -142,6 +155,17 @@ public class OidcJsonConfiguration {
 
     @JsonProperty("scope")
     protected String scope;
+    @JsonProperty("authentication-request-format")
+    protected String authenticationRequestFormat;
+
+    @JsonProperty("request-object-signing-algorithm")
+    protected String requestObjectSignatureAlgorithm;
+
+    @JsonProperty("request-object-encryption-algorithm")
+    protected String requestObjectEncryptAlgorithm;
+
+    @JsonProperty("request-object-content-encryption-algorithm")
+    protected String requestObjectContentEncryptionMethod;
```

Comment:
s/requestObjectContentEncryptionMethod/requestObjectContentEncryptionAlgorithm?

It's good for the variable name to be consistent with the configuration option.

Reply:
Just looking closer in the OIDC spec and I don't see references to "request object content encryption methods". Am just wondering if from a usability perspective, it will be confusing to determine what values should be specified for `request-object-encryption-algorithm` vs. `request-object-content-encryption-algorithm`. Do you recall the rationale for these names?

Reply:
Was just reviewing the analysis doc and see that we picked these names to match the Keycloak options in their console. However, it doesn't look like the OIDC spec uses the same terminology.

Reply:
Maybe we could use something like `request-object-encryption-enc-value` and `request-object-encryption-alg-value` to more closely align with the spec? WDYT?

Reply:
I think either could work. Initially in the proposal I did have `request-object-enc-value` following the naming from the openid metadata. 
However, in this conversation https://github.com/wildfly/wildfly-proposals/pull/532#discussion_r1307810834 we decided to switch to `request-object-content-encryption-algorithm`. This is more consistent with the keycloak console as you can see from the screenshot I attached on that conversation. 
Do you know which of the two (between the openID metadata and web console) the user is likely to use more? In that case, we can go with the naming convention for that. WDYT?

Reply:
Because OpenID providers other than Keycloak can be used, I think we should use names that align with the spec (sorry, I didn't realize that the Keycloak terminology was a bit different than what was in the spec initially!).

So I think request-object-encryption-enc-value and request-object-encryption-alg-value would be good.

Reply:
Okay, I will update the values. 
Thanks!